omero/sysadmins/unix/server-linux-walkthrough.txt

```diff
@@ -26,12 +26,17 @@ either use your own values, or alternatively source the following file:
 Prerequisites
 -------------
 
-As **root** install Ice, Java, Postgres:
+**The following steps are run as root.**
+
+
+Install Ice, Java, PostgreSQL:
 
 .. literalinclude:: walkthrough/step01_ubuntu1404_deps.sh
 
 | :download:`step01_ubuntu1404_deps.sh <walkthrough/step01_ubuntu1404_deps.sh>`
-| :download:`step01_centos6_deps.sh <walkthrough/step01_centos6_deps.sh>`
+| :download:`step01_centos7_deps.sh <walkthrough/step01_centos7_deps.sh>`
+
+.. note:: PostgreSQL needs to be reconfigured to accept TCP connections on CentOS 7.
```

Comment:
That's step is required in centos7
to make it clearer we will need a dedicated a centos7 as suggested for I can add in the note below the links ```see line in script``